src/webview.js

```diff
@@ -55,7 +55,7 @@ export default class WebView extends React.Component {
   }
 
   render() {
-    const tagProps = filterKeys(this.props, propName => propName in tagPropTypes)
+    const tagProps = filterKeys(this.props, propName => !(propName in eventPropTypes))
```

Comment:
Ah, interesting. It's probably better this way so we can still pass in `className`, `style`, etc.